.github/workflows/js-ci.yml

```diff
@@ -73,10 +73,10 @@ jobs:
         with:
           working-directory: js
 
-      - run: pnpm --filter=${{ env.WORKSPACE }} run lint
+      - run: pnpm --filter ${{ env.WORKSPACE }} run lint
```

Comment:
I cleaned up the way the filter is passed into the command, so it should now be the same everywhere.